package.json

```diff
@@ -13,20 +13,23 @@
 	"scripts": {
 		"build": "pnpm run policy-check && pnpm run layer-check && pnpm run generate:packageList && fluid-build --task build",
 		"build:compile": "fluid-build --task compile",
-		"build:docs": "fluid-build --task build:docs",
+		"build:docs": "fluid-build --task build:docs && npm run build:gendocs",
```

Comment:
To my knowledge, nothing was previously running `build:gendocs`, so I think my previous changes (no longer having each package copy its metadata report to the root) might have broken the part of our CI build that published the metadata artifact for consumption by the website build. This should fix it, though it's not clear to me if this is the "right" fix. @tylerbutler thoughts?

Reply:
Nope. This doesn't work.
https://dev.azure.com/fluidframework/public/_build/results?buildId=200139&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=09376e4d-15b7-5ca7-833f-f9b564c95cca